service/init.go

```diff
@@ -114,6 +116,11 @@ func (amf *AMF) Initialize(c *cli.Context) error {
 
 	amf.setLogLevel()
 
+	//Initiating a server for profiling
+	go func() {
+		http.ListenAndServe(":5001", nil)
```

Comment:
worried about hardcoding.  See example of gnbsim.. I believe we pass profilePort as arg.

Reply:
Shall make it config based